styles/styles.css

```diff
@@ -956,3 +960,25 @@ main .section.blue a.button.secondary::after {
     background-position: top right;
   }
 }
+
+.tag {
+  margin-left: 5px;
+  color: var(--background-color);
+  font-size: var(--body-font-size-xxxs);
+  font-weight: var(--font-weight-boldest);
+  line-height: 1;
+  border-radius: 3px;
+  padding: 2px 4px;
+}
+
+.tag-new {
```

Comment:
Is there going to be a limited number of tag names ? Or a limited number of tag colors ?

Reply:
I created a color for new and evolved since these are the tags that have a unique color any all of the other tags get the color of the improved tag since this one is not unique.